components/collection/utils/useCollectionDetails.ts

```diff
@@ -19,6 +21,7 @@ export type CollectionEntityMinimal = {
   currentOwner: string
   type: string
 }
+const chainsSupportingOffers = ['bsx']
```

Comment:
please use config map :) `*.config.ts`

Reply:
Could you point me to an example please?